asyncscope.md

```diff
@@ -2393,7 +2389,41 @@ creates an asynchronous operation (__[async.ops]__) that, when started:
 
 ## `execution::spawn`
 
-spec here
+Add the following as a new subsection immediately after __[exec.nest]__:
+
+::: add
+__`std::execution::spawn` [exec.scope.spawn]__
+
+[1]{.pnum} `spawn` associates the given input sender with the given token's async scope and, on success, eagerly starts the input sender.
```

Comment:
Since the association could fail, maybe this should say it "attempts" to associate…?
```suggestion
[1]{.pnum} `spawn` attempts to associate the given input sender with the given token's async scope and, on success, eagerly starts the input sender.
```